bedrock/contentful/api.py

```diff
@@ -23,10 +23,24 @@
 )
 from lib.l10n_utils import get_locale, render_to_string
 
-# Bedrock to Contentful locale map
-LOCALE_MAP = {
+# Bedrock and Contentful have slightly differing locale lists.
+BEDROCK_TO_CONTENTFUL_LOCALE_MAP = {
     "de": "de-DE",
+    "fr": "fr-FR",
+    "zh-TW": "zh-Hant-TW",  # NB: we should move to zh-Hant-TW in Bedrock - https://github.com/mozilla/bedrock/issues/10891
+    "nl": "nl-NL",
+    "id": "id-ID",
+    "it": "it-IT",
+    "jp": "ja-JP",
```

Comment:
This `"jp"` should be `"ja"`. We don't have any Japanese content in Contentful at the moment, so 've fixed it in the follow-up PR #11048